kge/config.py

```diff
@@ -746,6 +746,9 @@ def rename_value_re(key_regex, old_value, new_value):
                     renamed_keys.add(key)
         return renamed_keys
 
+    # 08.09.21
+    rename_key("entity_ranking.tie_handling", "entity_ranking.tie_handling.type")
```

Comment:
Does this work if one uses a config that already uses `entity_ranking.tie_handling.type`?

Reply:
I think this should work, at least we also did it for the optimizer like that:
```python
    # 09.10.20
    rename_key("train.optimizer", "train.optimizer.default.type")
    rename_keys_re("^train\.optimizer_args", "train.optimizer.default.args")
```